webpack/assets/javascripts/react_app/components/Permitted/Permitted.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useForemanPermissions } from '../../Root/Context/ForemanContext';
+
+/**
+ * Component to conditionally render a node if the current user has the requested permissions.
+ * Multiple permissions may be required by passing an array via **requiredPermissions**.
+ *
+ * Supply **requiredPermission** XOR **requiredPermissions**
+ * @param {string} requiredPermission: A single string representing a required permission
+ * @param {array<string>} requiredPermissions: An array of permission string.
```

Comment:
For a single permission, wouldnt it be easier to provide just one permission in an array?

Reply:
Yes, at second thought, having both options seems redundant and confusing. I will remove the single permission options.
Same goes for `Permitted`.